angles/test/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 if (CATKIN_ENABLE_TESTING)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
```

Comment:
This seems unrelated.

Reply:
Yes, this is indeed unrelated to the issue I wanted to solve. However, without it I was not able to build and run the tests. I felt it was alright to add this change here, but I can make a separate PR in case you think it would be more appropriate.

Reply:
Yes, please separate this. There's something else going on then in your workspace as this is passing CI w/o this change. Hard coding `-pthread` isn't a portable solution either.

Reply:
Ok, I will remove it and check what's wrong with my setup.